mailer/index.js

```diff
@@ -2,7 +2,7 @@ const EMAIL_CONTENT = {
   ACTIVATE_YOUR_ACCOUNT: {
     buttonText: 'Verify Email',
     contentTop:
-      'Your Ballers.ng account has been successfully created. To complete your registration, you need to confirm that we got your email address right.',
+      'Your ballers.ng account has been successfully created. Complete your registration by clicking on the link below to verify your email',
     subject: 'Verify your Email',
   },
   CHANGE_PASSWORD: {
```

Comment:
Lets make this be in past tense 🙏🏽 
```suggestion
  CHANGED_PASSWORD: {
```